examples/threathunter/threat_intelligence/README.md

```diff
@@ -126,7 +126,7 @@ An example of a custom Threat Intel connector that uses the `ThreatIntel` Python
 ## Troubleshooting
 
 ### Credential Error
-In order to use this code, you must have CBAPI installed and configured. If you receive the following message, visit the CBAPI GitHub repository for [instructions on setting up authentication](https://github.com/carbonblack/cbapi-python#api-token).
+In order to use this code, you must have CBAPI installed and configured. If you receive the following message, visit the CBAPI GitHub repository for [instructions on setting up authentication](https://developer.carbonblack.com/reference/carbon-black-cloud/authentication/).
```

Comment:
These needs a little more editing as you aren't redirecting to github. `CBAPI GitHub repository for` You may also want to mention a link to the how to configure the credential psc file

Reply:
Ah good catch here